src/components/GalleryPage/Winners.js

```diff
@@ -74,35 +75,41 @@ function Winners() {
 	}
 
 	const winnerCards = winners.map(item => {
+		const image = winnerImageMap.get(item.id);
 		return (
-			<Grid container key={item.title} spacing={2} alignItems='center'
-				style={{ paddingTop: 60 }}>
-				<Grid item md={5} xs={12}>
-					<Img fluid={winnerImageMap.get(item.id)} />
-				</Grid>
-				<Grid item md={7} xs={12}>
-					<Typography variant='h5'>{item.title} –{' '}
-						<span style={{ fontStyle: 'italic' }}>
-							{item.category}
-						</span>
+			<Grid item key={item.title} sm={6} style={{ paddingBottom: isSmall ? theme.spacing(4) : theme.spacing(8) }}>
+				<Box display='flex' flexDirection='column'>
+					<Img fluid={{ ...image, aspectRatio: 1.5 }} style={{ borderRadius: '14px', marginBottom: '1em' }}/>
+					<Typography variant='h5'>{item.title}</Typography>
+					<Typography variant='h6' style={{ fontWeight: theme.typography.fontWeightRegular,
+						textTransform: 'uppercase', fontSize: '1em', letterSpacing: '.5px' }}>
+						{item.category}
 					</Typography>
-					<Typography variant='body1' style={{ paddingBottom: 5, paddingTop: 5 }}>
+					<Typography variant='body1' style={{ padding: '0.5em 0em 1em' }}>
 						{item.description}
 					</Typography>
-					<Link href={item.link} target='_blank' rel='noreferrer noopener'>
-						See Project
-					</Link>
-				</Grid>
+					<a href={item.link} target='_blank' rel='noreferrer noopener' style={{ textDecoration: 'none' }}>
+						<Button variant='contained' disableElevation color="secondary"
+							style={{ textTransform: 'none', padding: '4px 1.5em' }}>
+							See Project
+						</Button>
+					</a>
```

Comment:
See  https://github.com/uclaacm/hoth.uclaacm.com/blob/fc6a4e1036cf531e5c325ee4ff0c205702d8514c/src/components/Footer/Footer.js#L70-L75 on how to make a button link.